js/mandala.js

```diff
@@ -0,0 +1,411 @@
+                'price': this.safeInteger (market, 'quotePrecision'),
+                'amount': this.safeInteger (market, 'basePrecision'),
+                'cost': this.safeInteger (market, 'basePrecision'),
+            },
+            'limits': {
+                'amount': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+                'price': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+                'cost': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+            },
+            'info': market,
+        };
+        return entry;
+    }
+
+    convertTrade (symbol, trade) {
+        const id = this.safeString (trade, 'id');
+        const timestamp = this.safeTimestamp (trade, 'time') / 1000;
+        const datetime = this.iso8601 (timestamp);
+        const price = this.safeFloat (trade, 'price');
+        const amount = this.safeFloat (trade, 'qty');
+        const order = '';
+        const type = 'limit';
+        const side = 'buy';
+        const cost = 0.00;
+        let takerOrMaker = undefined;
+        if (this.safeValue (trade, 'isBuyerMaker')) {
+            takerOrMaker = 'maker';
+        } else {
+            takerOrMaker = 'taker';
+        }
+        const entry = {
+            'info': trade,
+            'id': id,
+            'timestamp': timestamp,
+            'datetime': datetime,
+            'symbol': symbol,
+            'order': order,
+            'type': type,
+            'side': side,
+            'takerOrMaker': takerOrMaker,
+            'price': price,
+            'amount': amount,
+            'cost': cost,
+        };
+        return entry;
+    }
+
+    async fetchMarkets (params = {}) {
+        const method = 'openGetCommonSymbols';
+        const response = await this[method] (params);
+        const data = this.safeValue (response, 'data');
+        const markets = this.safeValue (data, 'list');
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const entry = this.convertSymbol (market);
+            result.push (entry);
+        }
+        return result;
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        const method = 'openGetMarketDepth';
+        await this.loadMarkets ();
+        const request = {
+            'symbol': this.markets[symbol]['id'],
+        };
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this[method] (this.extend (request, params));
+        return this.parseOrderBook (response);
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired ('fetchOrders requires a symbol argument');
```

Comment:
Can you as well add examples of symbols required, it makes for a clearer debugging.

Reply:
@mrbaguvix usually this is about all unified symbols, as documented in the docs: https://github.com/ccxt/ccxt/wiki/Manual#symbols-and-market-ids